### DISCUSSION TITLE
Temperature Heatmap

### DISCUSSION THREAD

**Discussion by zanac** [indomus/forum#2540] [GENERALE - Discussioni generiche]
Ciao,

vi segnalo la "preview" della mia prima card custom... ha molti bachi ancora, però inizia essere usabile!

https://github.com/zanac/temperature-heatmap-card

**Comment by zanac** [comment]
update: il repository è ora compatibile con HACS, aggiungete come risorsa esterna...!

**Comment by zanac** [comment]
online una nuova versione, ho aggiunto la possibilità di scorrere lo storico tramite due frecce
![image](https://github.com/indomus/forum/assets/21194919/aec25d91-ab93-4bf0-b47f-5a0cd1e750c9)

**Comment by dlio83** [comment]
Bellissima card complimenti!!! Utile ed intuitiva, ottimi tempi di risposta e configurazione immediata.

**Reply by zanac** [reply]
grazie! nel tempo aggiungerò altre piccole funzionalità.

**Comment by zanac** [comment]
update: ho aggiunto un editor grafico e alcune migliore nel look!

**Comment by zanac** [comment]
![image](https://github.com/indomus/forum/assets/21194919/233b4b00-0c7e-4bd0-a482-f70516a0a875)
In linea aggiornamento con icone trend!
